extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -61,6 +63,39 @@ static public function newFromId( $id, $master = false ) {
 		return null;
 	}
 
+	/**
+	 * @param array $ids
+	 * @return array
+	 */
+	static public function newFromIds( $ids ) {
+		wfProfileIn( __METHOD__ );
+		$test = "";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$test" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1481)